packages/portal/backend/src/controllers/PersonController.ts

```diff
@@ -59,6 +62,51 @@ export class PersonController {
         return successful;
     }
 
+    public async processClasslist(personId: string = null, path: string = null,  data: any): Promise<Person[]> {
+        Log.trace("PersonController::processClasslist(...) - start");
+
+        if (path !== null) {
+            data = await new CSVParser().parsePath(path);
+        }
+
+        this.duplicateDataCheck(data, ['ACCT', 'CWL']);
+        this.missingDataCheck(data, ['ACCT', 'CWL']);
+        const peoplePromises: Array<Promise<Person>> = [];
+
+        for (const row of data) {
+            // Log.trace(JSON.stringify(row));
+            if (typeof row.ACCT !== 'undefined' && typeof row.CWL !== 'undefined' &&
+                typeof row.SNUM !== 'undefined' && typeof row.FIRST !== 'undefined' &&
+                typeof row.LAST !== 'undefined') {
```

Comment:
MAJOR NOTE: 

@winstan I took out the LAB check, as students who are auditing the course or, perhaps, have not registered for a course yet may not have a LAB property.

Should we add it back in? Is it possible to remove these students from the `classlist` data that we are bringing in? 

Please let me know if you have a preference on how to proceed here.

Reply:
Let's not change any of the logic. If blank LAB was ignored before, ignore it here too.

Reply:
*LAB Check is back in*

We have agreed to take out the non-lab registrations in the classlist.